openprescribing/dmd2/build_search_filters.py

```diff
@@ -52,8 +52,17 @@ def _build_search_filter_fk(field):
     values = field.related_model.objects.values_list("cd", "descr").order_by("descr")
     values = [{r[0]: r[1]} for r in values]
 
+    # The type is "string", even though the values are actually integers.  This is
+    # because the QueryBuilder library calls parseInt on any values produced by a filter
+    # of type "integer" (see call to Utils.changeType in getRuleInputValue).  It turns
+    # out that parseInt cannot actually parse integers larger than
+    # Number.MAX_SAFE_INTEGER, which is (2 ** 53) - 1, or 9007199254740991, and loses
```

Comment:
:sob: 

Probably worth a bug report to QueryBuilder - they should assert something about the integer size?